### DISCUSSION TITLE
Want to reactivate Phone Link

### DISCUSSION THREAD

**Discussion by leo-mathurin** [ChrisTitusTech/winutil#1422] [Q&A]
Hey,

I know Microsoft Phone Link does eat a little bit of your cpu and ram but I really want to be able to use iMessages on my Windows 11 machine. 
Currently, after running the 'Laptop' tweaks as well as 'Set display for performance', 'Disable UAC' (I hate that Windows popup), and 'Set classic Right-Click menu'; the Phone Link tab in windows parameters is greyed out, and 'handled by my organization'.

Does anyone know how to reactivate this particular feature ?
Thank you for your time !

**Comment by leo-mathurin** [comment]
Any one have an idea to achieve this ?

**Comment by kennonb** [comment]
In the same boat. 😊

**Comment by Marterich** [comment]
Most likely this is a policy applied by the changes from[OO Shutup](https://www.oo-software.com/de/shutup10)
You could try downloading the program itself, export your current settings via File -> Export, and then revert any changes made by OO Shutup via Actions -> "Undo all changes"
Afterward, restart your computer and check if the problem persists.
It is always recommended to create a System restore point before changing anything in such a manner to have a fallback if anything were to go wrong. 
Also, you can go back to your previous config if you import and apply your previously exported config.

**Comment by LinuxOnTheDesktop** [comment]
What worked for me was the registry tweak (``) mentioned on [this Reddit page](https://www.reddit.com/r/Intune/comments/1fiw66w/enable_windows_11_phone_link/). I have not used 'WinUtil' though. (Rather I found this page by searching the web about a greyed-out option to enable PhoneLink.)
